qutebrowser/utils/urlutils.py

```diff
@@ -349,6 +337,40 @@ def raise_cmdexc_if_invalid(url):
         raise cmdexc.CommandError(get_errstring(url))
 
 
+def get_path_if_valid(pathstr, cwd=None, relative=False, check_exists=False):
+    """Check if path is a valid path.
+
+    Args:
+        pathstr: The path as string.
+
+    Return:
+        True if it is a valid path, False otherwise.
+    """
```

Comment:
Please adjust the docstring - it seems the description and return value are wrong, and not all arguments are documented.